### ISSUE TITLE
`Reused` label on subject and project cards does not meet minimum color contrast ratio threshold 

### ISSUE THREAD

**Issue by sudo-may** [NationalSecurityAgency/skills-service#2262]
![image](https://github.com/NationalSecurityAgency/skills-service/assets/51749745/9da89c64-d658-4381-a999-2b6eff1cc426)

Note: Please add cypress test to validate

**Comment by sudo-may** [comment]
PR: https://github.com/NationalSecurityAgency/skills-service/pull/2273

**Comment by dwalizer** [comment]
:+1:
